packages/server/src/middleware/error.js

```diff
@@ -5,8 +5,11 @@ import consola from 'consola'
 import Youch from '@nuxtjs/youch'
 
 export default ({ resources, options }) => function errorMiddleware(err, req, res, next) {
-  // ensure statusCode, message and name fields
+  if (/unexpected token/gi.test(err.message)) {
```

Comment:
This error is _originating_ from webpack. So would be better to handle in friendly-errors plugin.

Reply:
Here? https://github.com/nuxt/nuxt.js/blob/dev/packages/webpack/src/plugins/warnfix.js

Reply:
I will also recommend to add an url to the documentation and avoid `!` that feels frustrating.

Reply:
I mean add a similar transformer like this for `unexpectedToken.js`:

https://github.com/nuxt/friendly-errors-webpack-plugin/blob/master/src/transformers/moduleNotFound.js

And add it to [`defaultTransformers`](https://github.com/nuxt/friendly-errors-webpack-plugin/blob/master/src/friendly-errors-plugin.js#L12)

Reply:
Will do ☺️

Reply:
@manniL Can we close this now ?

Reply:
@clarkdo Feel free to close it, open an issue on https://github.com/nuxt/friendly-errors-webpack-plugin/ and assign me ☺️